layouts/section/meet-the-team.html

```diff
@@ -60,16 +63,470 @@ <h2 class="section--title">{{ i18n "key-contacts" }}</h2>
       {{ end }}
       {{ end }}
     </div>
-  </div>
 
-  <h2 class="section--title">{{i18n "meet-the-full-team"}}</h2>
+    <div class="heading"> 
+      <h2>{{ i18n "design" }}</h2>
+    </div>
+    <div class="row team-listing wb-eqht">
+      {{ range $member := .Site.Data.team.team.design  }}
+      {{ $v1 := not (isset $member "archived") }}
+      {{ if or ($v1) (eq $member.archived false) }}
+      <div class="col-xs-12 col-sm-6 col-md-3">
+        <div class="team-member">
+
+          <div class="photo-container">
+            <img class="lazy" src="/img/cds/placeholder.gif" data-src="{{ $baseURL}}{{ $member.imagehash }}.jpg" data-srcset="{{ $baseURL}}{{ $member.imagehash }}.jpg" alt="{{ index $member "name" }}">
+          </div>
+
+          <div class="text">
+            <span class="name">{{ $member.name }}</span>
+            <span class="title">{{ index $member.title (string ( $.Site.Language) ) }}</span>
+            <div class="social-links contact-info-social">
+              {{ if and (ne $member.email nil) (ne $member.email "") }}
+              <span class="social-icons">
+                <a href="mailto:{{ $member.email }}" aria-label="Link to {{ $member.name }}'s Email">
```

Comment:
Does the aria-label need to be translated?

Reply:
We don't have translations for this, this is from `main`

Reply:
It _should_ be translated though, should it? What does the aria-label say on the French site, is it in English?

Reply:
Yes, it's in English. Still says "Link to..."